docs/cmdline-opts/etag-compare.d

```diff
@@ -0,0 +1,17 @@
+Long: etag-compare
+Arg: <file>
+Help: Pass an ETag from a file as a custom header
+Protocols: HTTP
+---
+This option can be used to read an ETag from a file, and make
+an HTTP request conditional by sending a custom If-None-Match header
+using the extracted ETag.
+
+This option expects the first line of a file to be the desired ETag,
+everything after that is not considered. Empty files are not allowed.
+
+Use an option --etag-save to first save and ETag from response header,
```

Comment:
"Use the option --etag-save to first save the ETag from a response, and then use this option to compare using the saved ETag in a subsequent request."

Reply:
done